packages/experimental-identity-and-auth/src/middleware-http-signing/getHttpSigningMiddleware.ts

```diff
@@ -10,6 +10,7 @@ export const httpSigningMiddlewareOptions: FinalizeRequestHandlerOptions & Relat
   step: "finalizeRequest",
   tags: ["HTTP_SIGNING"],
   name: "httpSigningMiddleware",
+  aliases: ["apiKeyMiddleware", "tokenMiddleware", "awsAuthMiddleware"],
```

Comment:
For future eyes: `eventStreamHandlingMiddleware` (more likely this) and `eventStreamHeaderMiddleware` may need to be added to this list.